<launch>
    <!-- ros2 bag play /home/borges/humble_ws/src/ratslam/bags/irat_aus_28112011/irat_aus_28112011.db3 __clock -->
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="ratslam_view_template"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_MMC_lake_VO.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_pc"
        name="ratslam_pose_cells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_MMC_lake_VO.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_em"
        name="ratslam_experience_map"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_MMC_lake_VO.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_vo"
        name="ratslam_visual_odometry"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_MMC_lake_VO.yaml" />
    </node>     
            
</launch>

